src/background/actions/global-action-creator.ts

```diff
@@ -69,8 +69,8 @@ export class GlobalActionCreator {
         this.interpreter.registerTypeToPayloadCallback(Messages.UserConfig.GetCurrentState, this.onGetUserConfigState);
         this.interpreter.registerTypeToPayloadCallback(Messages.UserConfig.SetTelemetryConfig, this.onSetTelemetryConfiguration);
         this.interpreter.registerTypeToPayloadCallback(Messages.UserConfig.SetHighContrastConfig, this.onSetHighContrastMode);
-        this.interpreter.registerTypeToPayloadCallback(Messages.UserConfig.SetBugService, this.onSetBugService);
-        this.interpreter.registerTypeToPayloadCallback(Messages.UserConfig.SetBugServiceProperty, this.onSetBugServiceProperty);
+        this.interpreter.registerTypeToPayloadCallback(Messages.UserConfig.SetIssueService, this.onSetIssueService);
```

Comment:
Change to `SetIssueFilingService` to match updates elsewhere?

Reply:
About 120 instances of `IssueService` in code may want to be renamed to `IssueFilingService`